topology/probes/k8s/cache.go

```diff
@@ -41,16 +41,6 @@ import (
 type ResourceHandler interface {
 	Map(obj interface{}) (graph.Identifier, graph.Metadata)
 	Dump(obj interface{}) string
-	IsTopLevel() bool
```

Comment:
I added IsTopLevel at your request. In my previous implementation, the top level resources were passed as a string array. Just like in your proposition :-)

Reply:
@lebauce - you are correct - but when looking at the result I realized that there was no benefit to  using IsTopLevel() (for indicating a link the cluster) over the string array. And this became clearer when having to do the same with indicating l link to namespace(s).

Reply:
It was just a remark. I don't think the `IsTopLevel` is useful.